backend/games/builtin/heist/heist-runner.js

```diff
@@ -141,7 +141,7 @@ exports.triggerLobbyStart = (startDelayMins) => {
 
         const heistSettings = gameManager.getGameSettings("firebot-heist");
         const minTeamSize = heistSettings.settings.generalSettings.minimumUsers;
-        if (usersInHeist.length < minTeamSize) {
+        if (usersInHeist.length < minTeamSize - 1) { // user is added to usersInHeist after triggerLobbyStart is called in heist-command
```

Comment:
Just a heads up for the future, we try to keep our line lengths shorter than 120 characters. So we typically put code comments the line above the code. 

We don't have this documented anywhere or in our ES Lint rules so I realize theres no way you would've known this.